docs/polaris/data-science-workflows/applications/gpt-neox.md

```diff
@@ -0,0 +1,75 @@
+# Instructions for `gpt-neox`:
+
+1. Clone base environment from `conda/2022-07-19`:
+  ```shell
+  module load conda/2022-07-19
+  conda activate base
+  # NOTE: The path should be replaced in the --prefix flag below ⤵
+  conda create --clone base --prefix /path/to/conda/envs/gpt-neox-2022-07-19
```

Comment:
Would it better to add </path/to/conda> so that it throws an error and people are forced to change it in case they don't read it?

Reply:
Good call

Reply:
that would still be valid shell syntax